vllm/envs.py

```diff
@@ -24,6 +24,7 @@
     LOCAL_RANK: int = 0
     CUDA_VISIBLE_DEVICES: str | None = None
     VLLM_ENGINE_ITERATION_TIMEOUT_S: int = 60
+    VLLM_ENGINE_CORE_TIMEOUT_MS: int = 600000
```

Comment:
How about changing units to seconds and having the name reflect that it's the timeout for the engine(s) to become ready?
```suggestion
    VLLM_ENGINE_READY_TIMEOUT_S: int = 600
```

Reply:
Done